tests/templates/kuttl/tls/check-tls.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+import trino
+import argparse
+import sys
+import yaml
+
+
+if not sys.warnoptions:
+    import warnings
+warnings.simplefilter("ignore")
+
+
+def get_http_connection(host, user):
+    return trino.dbapi.connect(
+        host=host,
+        port=8080,
+        user=user,
+        http_scheme='http',
+    )
+
+
+def get_https_connection(host, user, verify):
+    return trino.dbapi.connect(
+        host=host,
+        port=8443,
+        user=user,
+        http_scheme='https',
+        verify=verify
+    )
+
+
+def get_authenticated_https_connection(host, user, password, verify):
+    return trino.dbapi.connect(
+        host=host,
+        port=8443,
+        user=user,
+        http_scheme='https',
+        auth=trino.auth.BasicAuthentication(user, password),
+        verify=verify
+    )
+
+
+def test_query(conn, query):
+    cursor = conn.cursor()
+    try:
+        cursor.execute(query)
+        result = cursor.fetchone()
+        print("[SUCCESS] Received: " + str(result))
+    except Exception as e:
+        print("[ERROR] " + str(e))
+        exit(-1)
+
+
+def test_query_failure(conn, query, failure_message):
+    cursor = conn.cursor()
+    try:
+        cursor.execute(query)
+        print("[ERROR] " + failure_message)
+        exit(-1)
+    except Exception as e:
+        print("[SUCCESS] Received expected exception: " + str(e))
+
+
+def read_yaml(config_path):
+    with open(config_path, 'r') as stream:
+        try:
+            config = yaml.safe_load(stream)
+        except yaml.YAMLError as e:
+            print("Could not load " + str(config_path) + ": " + str(e))
+            exit(-1)
```

Comment:
Isn't this what exceptions are for in the first place?

Reply:
Yeah but i prefer the clear message over that full stack trace.

Reply:
I guess "clear" is in the eye of the beholder, but stack trace line numbers can be very useful when the error is in the test itself.